src/boot/ual.ts

```diff
@@ -20,8 +24,132 @@ export default boot(({ app }) => {
+      accountName = localStorage.getItem('account');
+    } else {
+      //accountName = prompt('Input you account name', '');
+      await new Promise((resolve) => {
+        Dialog.create({
+          color: 'primary',
+          title: 'Connect to cleos',
+          message: 'Account name',
+          prompt: {
+            model: '',
+            type: 'text' // optional
+          },
+          cancel: true,
+          persistent: true
+        })
+          .onOk((data: string) => {
+            accountName = data != '' ? data : 'eosio';
+          })
+          .onCancel(() => {
+            //console.log('>>>> Cancel');
+            throw 'Canceled!';
+          })
+          .onDismiss(() => {
+            resolve(true);
+          });
+      });
+      await new Promise((resolve) => {
+        Dialog.create({
+          color: 'primary',
+          title: 'Connect to cleos',
+          message: 'Account permission',
+          options: {
+            type: 'radio',
+            model: [],
+            // inline: true
+            items: [
+              { label: 'Active', value: 'active' },
+              { label: 'Owner', value: 'owner' }
+            ]
+          },
+          cancel: true,
+          persistent: true
+        })
+          .onOk((data: string) => {
+            permission = data;
+          })
+          .onCancel(() => {
+            //console.log('>>>> Cancel');
+            throw 'Canceled!';
+          })
+          .onDismiss(() => {
+            resolve(true);
+          });
+      });
+    }
+    return {
+      accountName,
+      permission
+    };
+  }
+
+  async function signHandler(trx: string) {
+    const trxJSON: string = JSON.stringify(
+      Object.assign(
+        {
+          delay_sec: 0,
+          max_cpu_usage_ms: 0
+        },
+        trx
+      ),
+      null,
+      4
+    );
+    await new Promise((resolve) => {
+      Dialog.create({
+        color: 'primary',
+        message: `<pre>cleos -u https://${process.env.NETWORK_HOST} push transaction '${trxJSON}'</pre>`,
+        html: true,
+        cancel: true,
+        fullWidth: true,
+        ok: {
+          label: 'Copy'
+        }
+      })
+        .onOk(() => {
+          // console.log('OK')
+          copyToClipboard(
+            `cleos -u https://${process.env.NETWORK_HOST} push transaction '${trxJSON}`
+          )
+            .then((): void => {
+              Notify.create({
+                color: 'green-4',
+                textColor: 'white',
+                message: 'Copied to clipboard',
+                timeout: 1000
+              });
+            })
+            .catch(() => {
+              Notify.create({
+                color: 'red-8',
+                textColor: 'white',
+                message: 'Could not copy',
+                timeout: 1000
+              });
+            });
+        })
+        .onCancel(() => {
+          //console.log('>>>> Cancel');
+        })
```

Comment:
did you mean to `Throw 'Canceled!'` here? Can also remove commented out code

Reply:
Okay made the changes, also changed the spelling to American english XD @donnyquixotic